mer_verify_kernel_config

```diff
@@ -68,21 +116,33 @@ while (<>) {
   print "$conf matched, checking..." if $debug;
```

Comment:
I'll look into this.

Reply:
So collect valid rules and check them afterwards?

Reply:
I'd just remove the ones that don't apply. This could be cleaner, as you can do that in the block where you set $kernel_version. In fact, you won't need that variable outside of that block, which will make your PR a lot simpler. It also means there's no change to the logic if that's not defined.